src/app/utils/boosts.js

```diff
@@ -223,10 +226,17 @@ export const boostCombinations = [
             'multiplier'
           ) || 1
 
-        const boostMultiplier =
-          prismMultiplier * boostBoost * (1 + (artifactBoostBoostBonus / 100))
+          var boostMultiplier;
+
+          if(artifactBoostBoostBonus)
+            boostMultiplier = prismMultiplier * boostBoost * (1 + (artifactBoostBoostBonus / 100));
+          else
+            boostMultiplier = prismMultiplier * boostBoost;
+          
+
+        const dilMultiplier = dilithiumBoostBonus;
```

Comment:
nit: consider inlining this variable, dilMultiplier is used only once, so you could use dilithiumBoostBonus directly below.